apps/total-typescript/src/components/app/layout.tsx

```diff
@@ -70,6 +71,7 @@ const Layout: FunctionComponent<LayoutProps> = ({
         {children}
         {/* {footer ? footer : isNull(footer) ? null : <Footer />} */}
       </div>
+      <SurveyPopup />
```

Comment:
```suggestion
      {/* <SurveyPopup /> */}
```

This PR can be merged as is if we comment out this component.